docs/reference/tools-reference/llm-tool.md

```diff
@@ -50,7 +50,7 @@ Setup connections to provisioned resources in prompt flow.
 | frequency\_penalty     | float       | value that controls the model's behavior with regards to generating rare phrases. Default is 0.                             | No       |
 | best\_of               | integer     | the number of best completions to generate. Default is 1.                               | No       |
 | logit\_bias            | dictionary  | the logit bias for the language model. Default is empty dictionary.                     | No       |
-
+| user                   | string      | a unique identifier representing your end-user. Default is empty string.                | No       |
```

Comment:
this parameter is not visible in tool interface.

Reply:
deleted